cmd/apps/nginx_inc_app.go

```diff
@@ -0,0 +1,135 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallNginxIncIngress() *cobra.Command {
+	var command = &cobra.Command{
+		Use:          "nginx-inc",
+		Short:        "Install nginx-inc for OpenFaaS",
+		Long:         `Install nginx-inc for OpenFaaS`,
+		Example:      `arkade install nginx-inc`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("namespace", "n", "default", "The namespace used for installation")
+	command.Flags().Bool("update-repo", true, "Update the helm repo")
+	command.Flags().StringArray("set", []string{},
+		"Use custom flags or override existing flags \n(example --set key=value)")
+	command.Flags().String("controller-image", "", "Controller Image for NGINX Plus: (assuming you have pushed the Ingress controller image nginx-ingress to your private registry myregistry.example.com)")
+	command.Flags().Bool("plus", false, "Install Nginx Plus")
+	command.Flags().Int("replicas", 1, "The number of replicas of the Ingress controller deployment")
+	command.Flags().Bool("prometheus", false, "Install Nginx Plus")
+	command.Flags().Int64("prometheus-port", 9113, "Configures the prometheus port to scrape the metrics.")
+	command.Flags().String("kind", "deployment", "The kind of the Ingress controller installation - deployment or daemonset")
+
+	command.RunE = func(command *cobra.Command, args []string) error {
+		wait, _ := command.Flags().GetBool("wait")
+
+		updateRepo, _ := command.Flags().GetBool("update-repo")
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		namespace, _ := command.Flags().GetString("namespace")
+
+		clientArch, clientOS := env.GetClientArch()
+
+		fmt.Printf("Client: %s, %s\n", clientArch, clientOS)
+		log.Printf("User dir established as: %s\n", userPath)
+
+		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
+
+		overrides := map[string]string{}
+
+		nginxPlus, _ := command.Flags().GetBool("plus")
+		overrides["controller.nginxplus"] = fmt.Sprintf("%t", nginxPlus)
```

Comment:
There is a method in strconv that can do this.

Reply:
Done